docs/photutils/background.rst

```diff
@@ -158,13 +158,13 @@ mesh of a grid that covers the input data to create a low-resolution
 background map.  The final background or background rms map can then
 be generated by interpolating the low-resolution map.
```

Comment:
I guess this comment really belongs just a bit further up... But I wonder if you should add a short example in the previous section that demos all of the various Background (not 2D) classes?  Right now it all uses the sigma clipping functions, but now you can insert all of the various estimators implemented here.  Maybe this could just be a (logarithmic) histogram of the pixel values and then mark the output of the estimators as vertical lines?